File: ProjectFinalAdminBD/src/java/dominio/Usuario.java
package dominio;

/**
 * @author emfam
 */
public class Usuario {
    private String usuario;
    private String contrasenia;
    private int tipo;
    
    public Usuario(){}

    public Usuario(String usuario, String contrasenia, int tipo) {
        this.usuario = usuario;
        this.contrasenia = contrasenia;
        this.tipo = tipo;
    }

    public String getUsuario() {
        return usuario;
    }

    public void setUsuario(String usuario) {
        this.usuario = usuario;
    }

    public String getContrasenia() {
        return contrasenia;
    }

    public void setContrasenia(String contrasenia) {
        this.contrasenia = contrasenia;
    }

    public int getTipo() {
        return tipo;
    }

    public void setTipo(int tipo) {
        this.tipo = tipo;
    }

    @Override
    public String toString() {
        return "Usuario{" + "usuario=" + usuario + ", contrasenia=" + contrasenia + ", tipo=" + tipo + '}';
    }
    
}

File: ProyectoBD2021/domain/Usuario.php
<?php
	
	class Usuario
	{

		private $usuario;
		private $contrasena;
        private $tipo;

		function __construct($usuario,$contrasena,$tipo)
		{
			$this->usuario = $usuario;
			$this->contrasena = $contrasena;
            $this->tipo = $tipo;
		}


		public function getUsuario()
		{
			return $this->usuario;
		}

		public function getContrasena()
		{
			return $this->contrasena;
		}

        public function getTipo()
        {
            return $this->tipo;
        }

	}
 ?>